### PR TITLE
add ipyregulartable 0.1.3

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
  - [x] all JS licenses are generated with [`yarn licenses generate-disclaimer`](https://classic.yarnpkg.com/en/docs/cli/licenses/#toc-yarn-licenses-generate-disclaimer)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble be please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Some stuff upstream might color whether this version can be shipped:
- https://github.com/jpmorganchase/ipyregulartable/issues/23

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes ready for review!

File: recipes/ipyregulartable/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  noarch: python
+  script:
+    - {{ PYTHON }} -m pip install . -vv --no-deps
+    - cd js && yarn --production && yarn licenses generate-disclaimer > ../JS_LICENSES
+
+requirements:
+  build:
+    - yarn
+  host:
+    - jupyter-packaging
```
Comment (jakirkham) [inline recipes/ipyregulartable/meta.yaml:24]: Neat! Is this how we should be packaging Jupyter extensions going forward?

Comment (bollwyvl) [reply recipes/ipyregulartable/meta.yaml:24]: It's what the cookiecutters suggest, but has hidden gotchas.... see the `yarn` dependency above. once some of the `pyproject.toml` and `pip` stuff shakes out, i hope folks will be able to use `flit`, as it has a better (as in _an_) reproducible build story... but currently doesn't support `data_files`.

**Comment by bollwyvl** [comment]
Thanks!

File: recipes/ipyregulartable/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipes/ipyregulartable/meta.yaml:24]: Is it worth documenting this pattern in conda-forge for people to align on it? Or is it still too much of a moving target atm?

Comment (bollwyvl) [reply recipes/ipyregulartable/meta.yaml:24]: Nah... `grayskull` picked up the `setup_requires` just fine... the upstream issue pointed out the piece about `js` not being prebuilt in the `sdist`.

**Pull request merged**